notebooks/pokemon_classifier.ipynb

```diff
@@ -592,8 +582,8 @@
     "            dst = os.path.join(output_directory, pokemon_id + \".png\")\n",
     "            shutil.copyfile(src, dst)\n",
     "\n",
-    "            # Set image path in data frame\n",
-    "            pokemon.loc[pokemon[\"pokedex_number\"] == int(pokemon_id), 'imagePath'] = dst\n",
+    "    # Set image path in data frame\n",
+    "    pokemon['imagePath'] = pokemon['pokedex_number'].apply(lambda x: os.path.join(output_directory, str(x).zfill(3) + '.png'))\n",
```

Comment:
This should be more efficient as you do not have to call `.loc` and value assignment for every file in the folder. In this case, it is possible to apply the same function for the whole column.